### DISCUSSION TITLE
能否支持代理后通过ipv6访问存储商

### DISCUSSION THREAD

**Discussion by LSP0123** [AlistGo/alist#4099] [Ideas]
本人家庭带宽100兆，但是ipv6可以跑到1000兆
我在使用alist挂载了百度网盘，部分文件下载可以跑到800+mbps，应该是访问了百度的ipv6地址。
请问能否通过指定alist在访问存储商网盘时访问ipv6来提高速度？

**Comment by skyflying1989** [comment]
抓下网盘下载的网址链接是哪个，然后用host文件绑定对应的ipv6地址即可。。。
